src/cpp/rtps/builtin/discovery/database/DiscoveryParticipantsAckStatus.hpp

```diff
@@ -45,15 +45,25 @@ class DiscoveryParticipantsAckStatus
 
     ~DiscoveryParticipantsAckStatus() = default;
 
+    enum class ParticipantState : uint8_t
```

Comment:
What do you think about changing the naming here to
```
enum class DiscoveryInfoState
{
  PENDING_SEND, // Data(P) ....
  WAITING_ACK,
  ACKED
};
```

I think it could be more congruent with the context